muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/actions/workspaces/material.ts

```diff
@@ -904,13 +905,43 @@ const updateWorkspaceMaterialContentNode: UpdateWorkspaceMaterialContentNodeTrig
           throw err;
         }
 
+        let isConflictError = false;
+
+        // The "message.reason === "CONTAINS_ANSWERS"" is only available for admins, who receive a conflict error (409),
+        if (err.message) {
+          try {
+            const message = JSON.parse(err.message);
+            if (message.reason === "CONTAINS_ANSWERS") {
+              isConflictError = true;
+            }
+            // eslint-disable-next-line no-empty
+          } catch (e) {}
+        }
```

Comment:
Is this try catch block needed? It doesn't have any effect to anything at the moment